flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/zookeeper/ZooKeeperHaServicesTest.java

```diff
@@ -224,23 +224,26 @@ private void runCleanupTestWithJob(
             final RunningJobsRegistry runningJobsRegistry =
                     zooKeeperHaServices.getRunningJobsRegistry();
 
-            final LeaderRetrievalUtils.LeaderConnectionInfoListener listener =
+            final LeaderRetrievalUtils.LeaderConnectionInfoListener resourceManagerLeaderListener =
                     new LeaderRetrievalUtils.LeaderConnectionInfoListener();
-            resourceManagerLeaderRetriever.start(listener);
+            resourceManagerLeaderRetriever.start(resourceManagerLeaderListener);
             resourceManagerLeaderElectionService.start(
                     new TestingContender("foobar", resourceManagerLeaderElectionService));
             LeaderElectionService jobManagerLeaderElectionService =
                     zooKeeperHaServices.getJobManagerLeaderElectionService(jobId);
             jobManagerLeaderElectionService.start(
-                    new TestingContender("", jobManagerLeaderElectionService));
+                    new TestingContender("barfoo", jobManagerLeaderElectionService));
```

Comment:
```suggestion
                    new TestingContender("unused-jobmanager-address", jobManagerLeaderElectionService));
```
nit: I understand that you just followed the scheme of the other `TestingContender`. But I usually find it better to make the purpose of a certain String parameters visible if possible. That improves the readability of the test code. Without an IDE, I wouldn't be able to see right away what this parameter is used for if I'm not familiar with the `TestingContender` class.

Reply:
Sounds good. I will change it.